parsers/CA_AB.py

```diff
@@ -79,9 +79,12 @@ def fetch_price(
     prices = PriceList(logger)
     for row in csv.reader(response.text.split("\r\n\r\n")[2].splitlines()[1:]):
         if row[1] != "-":
+            date, hour = row[0].split()
             prices.append(
                 zoneKey=zone_key,
-                datetime=arrow.get(row[0], "MM/DD/YYYY HH", tzinfo=TIMEZONE).datetime,
+                datetime=datetime.strptime(
+                    f"{date} {int(hour) - 1}", "%m/%d/%Y %H"
+                ).replace(tzinfo=TIMEZONE),
```

Comment:
The date no longer matches the old version and is off by 1 hour, I'll take a deeper look at it later today but I think you need to add back one hour using `timedelta()` after it has been converted to a datetime.

Reply:
I see that now. My previous test must have been in a time window that didn't show the >2h difference. I also looked back at PR #6179 and it didn't seem clear if timedelta would be need or not - it looks like it is. I have corrected it with timedelta.